docs/install.md

```diff
@@ -32,8 +32,15 @@ Kubernetes](#installation-and-setup).
 
 The administrator must install PMEM-CSI, using [the PMEM-CSI
 operator](#install-using-the-operator) (recommended) or with [scripts
-and YAML files in the source code](#install-from-source). A PMEM-CSI
-installation can only use [direct device
+and YAML files in the source code](#install-from-source). The default
+install settings should work for most clusters. Some clusters don't
+use `/var/lib/kubelet` as the data directory for kubelet and then the
+corresponding PMEM-CSI setting must be changed accordingly because
+otherwise kubelet does not find PMEM-CSI. The operator has an option
+for that in its API, the YAML files can be edited or modified with
```

Comment:
Would be nice to add a reference to operator API here.

Reply:
Added.